model/degree.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Degree struct {
```

Comment:
Fields should be like 
Degree Name , Type , Duration , start , end , percentage